noria/Cargo.toml

```diff
@@ -44,6 +44,7 @@ tracing-futures = "0.2.2"
 slab = "0.4"
 pin-project = "0.4.0"
 futures-util = "0.3.0"
+mysql_common = "0.20.2"
```

Comment:
We really ought to go through and make some of these features optional and guarded by features. But I think it's fine for now.